packages/spor-elm/src/Spor/TravelTag.elm

```diff
@@ -0,0 +1,244 @@
+module Spor.TravelTag exposing (..)
+
+{-| A component for displaying travel tags
+
+
+## Config
+
+@docs init, withVariant, withSize, withTitle, withDescription, withColor, withBackgroundColor
+
+
+## Display
+
+@docs toHtml
+
+-}
+
+import Css exposing (Color)
+import Html.Styled as Html exposing (Html)
+import Html.Styled.Attributes as Attributes
+import Spor.Common.Types exposing (Size(..), Variant(..))
+import Spor.LineTagIcon as LineTagIcon
+import Spor.LineTagText as LineTagText
+import Spor.Token.Color.Alias as Alias
+import Spor.Token.Color.Linjetag as Linjetag
+import Spor.Token.Size.Spacing as Spacing
+
+
+{-| A component for displaying travel tags
+-}
+type TravelTag
+    = TravelTag Options
+
+
+type alias Options =
+    { variant : Variant
+    , size : Size
+    , title : String
+    , description : Maybe String
+    , backroundColor : Maybe Color
+    , color : Maybe Color
+    }
```

Comment:
For avvik kalte jeg APIet `deviationLevel`, og da den enumverdiene "critical", "major", "minor", "info" og "none" (none er default).

Reply:
Tenker å få det med i neste fletteforespørsel